macro/src/main/scala/SparkSchemaHelper.scala

```diff
@@ -51,16 +51,14 @@ class SparkSchemaHelperImpl(val c: Context) {
   def fromRow[T: c.WeakTypeTag](row: c.Expr[Row]): c.Expr[T] = {
     import c.universe._
     val optionType = typeOf[Option[_]]
-    val optionSeq = typeOf[Seq[Option[_]]]
+    val optionSeq = typeOf[Iterable[Option[_]]]
```

Comment:
I think the fix is usually to import `immutable.Seq` explicitly? https://docs.scala-lang.org/overviews/core/collections-migration-213.html